packages/devtools_app/lib/src/logging/logging_controller.dart

```diff
@@ -215,27 +227,36 @@ class LoggingController {
 
   void _updateStatus() {
     final int count = _loggingTableModel.rowCount;
-    final String label = count >= kMaxLogItemsLowerBound
-        ? '${nf.format(kMaxLogItemsLowerBound)}+'
-        : nf.format(count);
+    final String label = count >= kMaxLogItemsLowerBound ? '${nf.format(kMaxLogItemsLowerBound)}+' : nf.format(count);
     onLogCountStatusChanged('$label events');
   }
 
   void clear() {
     data.clear();
+    rawData.clear();
+    detailsController?.setData(null);
+    _loggingTableModel.setRows(data);
+  }
+
+  void search(String text) {
+    searchText = text;
+    data = rawData.where((item) {
+      if (!useSummaryFilter) {
```

Comment:
what is the use case for sometimes only searching the summary and sometimes searching the summary and details?